### PR TITLE
DOC: add plot_events to example

### LINKED ISSUE #2775: Use plot_event in a example
The function `plot_event` has currently no example linked to its [doc](https://nilearn.github.io/modules/generated/nilearn.plotting.plot_event.html#nilearn.plotting.plot_event). 
It wouldn't be too costly to use it in one example somewhere.

### PR DESCRIPTION
**Pull request by Craskermasker**
Closes #2775  .

Changes proposed in this pull request:

- add `plot_events` to example

### CODE REVIEW

File: examples/04_glm_first_level/write_events_file.py [hunk 1]
```diff
@@ -63,3 +63,10 @@
 tsvfile = 'localizer_events.tsv'
 events.to_csv(tsvfile, sep='\t', index=False)
 print("Created the events file in %s " % tsvfile)
+
+#########################################################################
+# Optionally, the events can be visualized using the plot_event function.
+from matplotlib import pyplot as plt
+from nilearn.plotting import plot_event
+plot_event(events, figsize=(15,5))
```
Comment (bthirion) [inline examples/04_glm_first_level/write_events_file.py:71]:
```suggestion
plot_event(events, figsize=(15, 5))
```

**Review by bthirion — COMMENTED** [review]
LGTM otherwise. Thx !

**Review by bthirion — APPROVED** [review]
LGTM, thx.

**Review by NicolasGensollen — APPROVED** [review]
LGTM! Thanks @Craskermasker !

**Pull request merged**